### DISCUSSION TITLE
Transparency setting for enemies that are casting interruptible spells

### DISCUSSION THREAD

**Discussion by jvalrog** [Backupiseasy/ThreatPlates#494] [Ideas]
Right now the "Enemy Casting" setting will apply to any cast. Many times in a trash group, many units will start casting uninterruptible spells. In that case this setting is a bit useless to me.

I'd love to make 0% transparent those units that are casting only interruptible spells so it's more easy to spot them.

Don't know if it's possible though

**Comment by Backupiseasy** [comment]
There have been feature requests that go into the same direction:

* https://github.com/Backupiseasy/ThreatPlates/issues/258
* https://github.com/Backupiseasy/ThreatPlates/issues/195
* https://github.com/Backupiseasy/ThreatPlates/issues/257

It's definitely something that can be implemented. I also would not be that difficult; I also can understand the reasoning - in mob groups it's often very difficult to see the cast you want to see and interrupt. 

Hiding all non-interruptible casts, on the other hand, might also not be good as some casts might require a reaction from you, e.g., run away.
